packages/components/Organisms/CampaignSecondary/index.vue

```diff
@@ -71,6 +72,9 @@ export default {
 
   .rpl-campaign-secondary {
     position: relative;
+    @include rpl_breakpoint('xs') {
```

Comment:
xs is 0 so, this could be written just as:
`padding-bottom: $rpl-campaign-secondary-padding-xs;`